pkg/shares/non_interactive_defaults.go

```diff
@@ -83,10 +83,31 @@ func roundUpBy(cursor, v int) int {
 	}
 }
 
-// MinSquareSize returns the minimum square size that can contain shareCount
-// number of shares.
-func MinSquareSize(shareCount int) int {
-	return RoundUpPowerOfTwo(int(math.Ceil(math.Sqrt(float64(shareCount)))))
+// SubTreeWidth determines how many shares should be included when creating
+// subtree. This is used for determining which subtree roots are used in the
+// share commitment for PFBs, and therefore also the padding between blobs. The
+// reasoning behind this algorithm is discussed in depth in ADR013.
```

Comment:
Would include a link here as well if possible

Reply:
for now, I have only included `celestia-app/docs/architecture/adr-013...` instead of a link, because I don't want to use an outdated or relative link

however, I did leave a note in #1607 to update links here after that or the changes to the specs are added